src/packs.c

```diff
@@ -270,6 +270,8 @@ int download_subscribed_packs(struct list *subs, struct manifest *mom, bool requ
 		/* no packs needs to be downloaded */
 		info("No packs need to be downloaded\n");
 		progress_complete_step();
+		progress_set_next_step("pack_extraction");
+		progress_complete_step();
```

Comment:
why are we showing pack_extraction step if there were no packs downloaded? for consistency so we have the same number of steps regardless of if we download packs or not?

Reply:
yes. just for consistency. I'll change that to an empty step

Reply:
good